README.md

```diff
@@ -3,3 +3,60 @@
+| View public scenarios                                                                       |   ✅   |   ✅   |   ✅   |   ✅   |
+| Modify unlocked global inputs in unlocked <br> global scenarios without app storing values  |   ✅   |   ✅   |   ✅   |   ✅   |
+| About modal                                                                                 |   ❌   |   ✅   |   ✅   |   ✅   |
+| Simulations dropdown                                                                        |   ❌   |   ✅   |   ✅   |   ✅   |
+| Download sims                                                                               |   ❌   |   ✅   |   ✅   |   ✅   |
+| Scenarios dropdown                                                                          |   ❌   |   ✅   |   ✅   |   ✅   |
+| Bookmarks panel                                                                             |   ❌   |   ❌   |   ✅   |   ✅   |
+| Create scenarios                                                                            |   ❌   |   ❌   |   ✅   |   ✅   |
+| Edit scenarios                                                                              |   ❌   |   ❌   |   ✅   |   ✅   |
+| Delete scenarios                                                                            |   ❌   |   ❌   |   ✅   |   ✅   |
+| Duplicate scenarios                                                                         |   ❌   |   ❌   |   ✅   |   ✅   |
+| Run sims                                                                                    |   ❌   |   ❌   |   ✅   |   ✅   |
+| Delete sims                                                                                 |   ❌   |   ❌   |   ✅   |   ✅   |
+| Edit themes                                                                                 |   ❌   |   ❌   |   ✅   |   ✅   |
+| Copy/Delete Attributes                                                                      |   ❌   |   ❌   |   ✅   |   ✅   |
+| Delete owned bookmarks                                                                      |   ❌   |   ❌   |   ✅   |   ✅   |
+| Remove schema edits                                                                         |   ❌   |   ❌   |   ❌   |   ✅   |
+| Emulate permissions                                                                         |   ❌   |   ❌   |   ❌   |   ✅   |
+| Export non-Simulation layers                                                                |   ❌   |   ❌   |   ❌   |   ✅   |
+| Configurer modal                                                                            |   ❌   |   ❌   |   ❌   |   ✅   |
+| Download all scenarios                                                                      |   ❌   |   ❌   |   ❌   |   ✅   |
+| Show archived filters                                                                       |   ❌   |   ❌   |   ❌   |   ✅   |
+| Archive/unarchive filters                                                                   |   ❌   |   ❌   |   ❌   |   ✅   |
+
```

Comment:
Admins have the ability to do things in the "Lab Settings" of the config that could be enumerated here.

In particular, the "Advanced Settings" give the admin the ability to hide certain elements for non-admins.  If non-admins typically have the ability to do something but it can be turned off by an admin, that is something that should be distinguished here. For example, those could have a yellow check box that indicates it might not be available if that's been disabled by an admin.